app/src/main/res/values/component_colors.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="label_text_color">@color/primary_text_color</color>
```

Comment:
This is rather generic and feels like something that would go in color_palette, not component_colors. Ditto for the next few.

I'm actually thinking that component colors should be shared very little outside of their respective views/fragments/activities.

Reply:
I tried restructuring `component_colors` into something more restricted into the scope of their activities/views only. PTAL now. If something is still off can you please provide examples for how it should be.